gammapy/irf/effective_area.py

```diff
@@ -0,0 +1,87 @@
+#Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+from ..utils.nddata import NDDataArray, DataAxis, BinnedDataAxis
+from collections import OrderedDict
+import numpy as np
+
+class EffectiveArea2D(NDDataArray):
+    """2D effective area table
+
+    Axes: ``THETA``, ``ENERG``
+
+    Parameteres
+    -----------
+    energy : `~astropy.units.Quantity`, `~gammapy.utils.BinnedDataAxis`
+        Energy axis (binned)
+    offset : `~astropy.units.Quantity`, `~gammapy.utils.DataAxis
```

Comment:
Missing backtick at the end of the line.

Can you please check the HTML docs for the new classes in this PR locally?